Gems/Atom/Feature/Common/Code/Source/Material/MaterialConverterSystemComponent.cpp

```diff
@@ -183,6 +187,13 @@ namespace AZ
                 sourceData.SetPropertyValue(Name{"roughness.factor"}, roughness.value());
             }
 
+            if (!materialData.GetRoughnessFactor().has_value() && abs(materialData.GetShininess()) > AZ::Constants::FloatEpsilon)
+            {
+                // When the MaterialData provides Shininess instead of Roughness, it is necessary to convert Shininess to Roughness.
```

Comment:
```suggestion
            }
            else if (abs(materialData.GetShininess()) > AZ::Constants::FloatEpsilon)
            {
                // When the MaterialData provides Shininess instead of Roughness, it is necessary to convert Shininess to Roughness.
```